aea/crypto/helpers.py

```diff
@@ -93,25 +103,94 @@ def _try_validate_ethereum_private_key_path(
             raise
 
 
-def _validate_private_key_path(private_key_path: str, ledger_id: str):
+def _try_validate_cosmos_private_key_path(
```

Comment:
We should try to get rid of the helpers and just expose these methods on the crypto object @marcofavorito ...load and dump already implement most of the functionality